pipeline/pipeline-notes.rst

```diff
@@ -22,37 +22,71 @@ Make sure you edit your security groups to include port 22 (SSH) and port
 80 (HTTP) ; you'll need the first one to log in, and the second one to 
 connect to the ipython notebook.
 
-Once you ssh in, set yourself up to run as root.::
+We tested this script on a RackSpace instance with specifications::
+  #TODO get rackspace specifications
+If you are running on RackSpace, we may need to add extra space for data by formating and mounting a drive.
+Formating/Mounting Drive::
+  mkfs -t ext4 /dev/xvde1
+  mkdir /mnt/data
+  mount /dev/xvde1 /mnt/data/ -t ext4
+  # To make a regular user 
+  adduser non-root # TODO find quick setup instructions
 
- sudo su
+Additionally, if you are on RackSpace, you will need to disable the firewall so that you can run the iPython notebook.::
 
-Next, we are going to set the instance up with many of the software 
-packages we will need::
+  sudo service ufw stop
 
+Next, we are going to set the instance up with many of the software 
+packages we will need. You will need root permissions to install these::
 
- apt-get update
- apt-get --yes install screen git curl gcc make g++ python-dev unzip \
+ sudo apt-get update
+ sudo apt-get --yes install screen git curl gcc make g++ python2.7-dev unzip \
             default-jre pkg-config libncurses5-dev r-base-core \
             r-cran-gplots python-matplotlib sysstat bowtie \
             texlive-latex-recommended mummer python-pip ipython \
-            ipython-notebook bioperl ncbi-blast+
+            ipython-notebook bioperl ncbi-blast+ python-virtualenv hmmer \
+            ncbi-tools-bin prodigal infernal aragorn parallel
 
+You'll also need this because reasons::
 
-Now, you'll need to install the version of 'khmer' that the
+
+Now, with root privledges, you'll need to install the version of 'khmer' that the
 paper is currently using.::
  
+ virtualenv venv
+ source venv/bin/activate
  easy_install -U setuptools
  pip install khmer==1.1
+ pip install --upgrade numpy
```

Comment:
Wrong URL, we're running Ubuntu Trusty:
http://packages.ubuntu.com/trusty/python-numpy

On Wed, Feb 11, 2015, 10:47 Michael Crusoe <michael.crusoe@gmail.com> wrote:

> Is the python-numpy Debian package new enough to work?
> http://packages.ubuntu.com/utopic/python-numpy
>
> On Tue, Feb 10, 2015, 23:40 Michael Wright <notifications@github.com>
> wrote:
>
>> In pipeline/pipeline-notes.rst
>> <https://github.com/ged-lab/2012-paper-diginorm/pull/6#discussion_r24453165>
>> :
>>
>> >   easy_install -U setuptools
>> >   pip install khmer==1.1
>> > + pip install --upgrade numpy
>>
>> Added this line due to:
>>
>> (venv)non-root@cloud-server-02:/mnt/data/2012-paper-diginorm/pipeline$
>> make
>> python calc-r2.py > calc-r2.txt
>> Traceback (most recent call last):
>> File "calc-r2.py", line 1, in <module>
>> import numpy, math, itertools
>> ImportError: No module named numpy
>> make: *** [calc-r2.txt] Error 1
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/ged-lab/2012-paper-diginorm/pull/6/files#r24453165>.
>>
>